### PR TITLE
Add dockerfile and make commands

### PR DESCRIPTION
**Pull request by siminm**

### CODE REVIEW

**Comment by siminm** [comment]
@diranged this is ready for review
image

cc: @nrvale0

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+FROM alpine:3.2
+MAINTAINER Mikhail Simin <mikhail@nextdoor.com>
+
+RUN mkdir -p /app /app/zk_monitor
+ADD zk_monitor/runserver.py /app/zk_monitor/runserver.py
```
Comment (diranged) [inline Dockerfile]: Umm .. this doesn't really do much though. You've added runserver.py, but *none of the other code*.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,15 @@
+FROM alpine:3.2
+MAINTAINER Mikhail Simin <mikhail@nextdoor.com>
+
+RUN apk add --update bash python py-pip py-yaml ca-certificates curl wget
```
Comment (diranged) [inline Dockerfile]: Remove the copy-pasted `curl wget`

File: Makefile.docker [hunk 3]
```diff
@@ -0,0 +1,82 @@
+# Standard settings that will be used later
```
Comment (diranged) [inline Makefile.docker]: Remove this file

File: sample_config.yaml [hunk 4]
```diff
@@ -0,0 +1,10 @@
+# Example Service
+/services/path/to/service:
+  alerter:
+    email:
+     - eng+zkmonitor@nextdoor.com
```
Comment (diranged) [inline sample_config.yaml]: Yank out this email address..

**Comment by diranged** [comment]
Just a few bits.. then looks good! :+1:

File: entrypoint.sh [hunk 5]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+DOCKER_HOST_IP=$(route -n | awk '/UG[ \t]/{print $2}')
+ZOOKEEPER_HOST=${ZOOKEEPER_HOST:-$DOCKER_HOST_IP}
+ZOOKEEPER_PORT=${ZOOKEEPER_PORT:-2181}
+VERBOSE=${VERBOSE:-}
+
+CONFIG_FILE='./sample_config.yaml'
+if [ - ./config.yaml ]; then
```
Comment (diranged) [inline entrypoint.sh]: Remove this if statement. Change to:

```
...
LOGLEVEL=${LOGLEVEL:-info}
CONFIG_FILE=${CONFIG_FILE:-/config.yaml}
zk_monitor -z $ZOOKEEPER_HOST:$ZOOKEEPER_PORT -l $LOGLEVEL -f $CONFIG_FILE
```

**Comment by diranged** [comment]
:+1:

**Pull request merged**